client/sections.js

```diff
@@ -562,6 +562,12 @@ const sections = [
 		module: 'calypso/my-sites/promote-post',
 		group: 'sites',
 	},
+	{
+		name: 'subscriptions',
+		paths: [ '/subscriptions' ],
+		module: 'calypso/my-sites/subscriptions',
+		group: 'sites',
```

Comment:
Just want to add that `enableLoggedOut: true` should make it work for logged out users (will still need to do cookie auth) separately.

And `isomorphic: true` makes this section server-side rendered (not sure if we'll need them).